unitTests/ui/LiveAnalyticsClientTest.ts

```diff
@@ -91,7 +91,7 @@ export function LiveAnalyticsClientTest() {
             queryText: 'the query',
             advancedQuery: 'the advanced query',
             didYouMean: true,
-            numberOfResults: 4,
+            numberOfResults: 3,
```

Comment:
I'm not sure why `4` was needed here. The query requests 10 results, and the `promise` returns 3 results.